source/_footer.erb

```diff
@@ -1,5 +1,4 @@
-<hr class="hidden" />
-<div id="footer">
-  <p><small><%= partial 'license' %></small></p>
-  <p><small>Powered by <a href="http://middlemanapp.com/">Middleman</a>.</small></p>
-</div>
+<footer id="footer">
```

Comment:
Should we remove the redundant `id`s too?  I can't imagine needing a 2nd `<footer>` element so it should be safe to rewrite any CSS selectors from `#footer { ... }` to `footer { ... }` right?  Same goes for `header`.

I'm wary of pushing too much into one PR though and exhausting your patience.  So maybe this is one for a future PR - there's a _lot_ of `id` based styling in here that could probably be class or element based.

Reply:
From what I understanding styling on the tag name isn't recommended, neither is the ID.

My intension is to create another PR where I switch the styling to use an approach like `.footer`/`.footer__block--modifier` ;)

Reply:
Let's not use BEM, it's gross.  Seems weird that "best practice" would push us to do `<footer class="footer">` as the `class` is clearly redundant.  But 🤷.

Reply:
Oh no! I love BEM, but I can compromise on how we name the classes :)

Though the main reason I really like using classes for styling, is it'll mean we have a consistent level of [CSS specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), which'll be less surprises to anymore else wanting to jump on board :)

Reply:
Hmm, it's not really a hill I want to die on, and you're doing the work so it's churlish of me to impose my antiquated reckons on it.  It just seems weird to me is all, and I am certainly not a frontend specialist these days so someone closer to things should probably make the call.  Might be willing to die on the anti-BEM hill though 😉